docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc

```diff
@@ -1,43 +1,77 @@
-{PROJECT_NAME} leverages commoditized mobile hardware, making use of each employee’s mobile phone.
+- Mobile worker (operator) application
+- Management Portal (Backoffice) application
 
-The primary goal of {PROJECT_NAME} is to ease mobile enablement efforts.
-It achieves that by providing set of modules that can be integrated into existing applications additionally
-{PROJECT_NAME} allows developers to leverage existing demo-solution by extending it to suit their business needs.
+Both applications are utilizing Node.js based server by utilizing restfull api.
 
-Main features in the latest {WFM-RC-Version}:
+== Goals
 
-* Workflow Management
-* Real-time Data Access
-* Data Synchronization
+The primary goal of {PROJECT_NAME} is to minimize development effort required to build WFM applications. It achieves that by providing set of modules that can be integrated into existing applications.
+Developers can effectively reuse them in their applications or base on {PROJECT_NAME} reference applications that integrate all modules into fully functional solution. {PROJECT_NAME} targets to satisfy requirements that each production website and mobile application needs.
+In particular:
 
-Roadmap:
+- Mobile development (using Apache Cordova plugins)
+- Real-time Data Access and synchronization
+- Authentication and Authorization (using multiple providers)
+- Security (using best patterns for mobile and website security)
+- Logging (tracing issues out of the box)
+- User interface (AngularJS based interface that can be customized)
 
-* Plugable authentication
-* Image capture
-* Geolocation
-* Internal Messaging and 3rd party Messaging
-* Mobile Forms
-* Real-time Data Access
-* Integration of sensor data
+It allows developers to leverage production hardened applications by extending them to suit their business needs. Each application can be customized by writing or reusing one of the available {PROJECT_NAME} modules.
 
-{PROJECT_NAME} {WFM-RC-Version} features:
+{PROJECT_NAME} separates modules into different categories:
+
+* Core modules containing main functionality (@raincatcher/*)
+** Client modules used in mobile and portal applications
+** Server modules to be used only in server application
+* User interface modules  (@raincatcher/angularjs-*)
+* Step modules (@raincatcher/step-*)
+* Example modules (@raincatcher-examples/*)
+* External modules build by RainCatcher users and custom code in reference applications
+
+image:diagrams/customSolution.png[customSolution]
+
+== Process Management
+
+{PROJECT_NAME} focuses mainly on diginalizing business process
+by including set of the modules responsible for the process managment to the user applications. Process managment framework can be seen as simplified version of the BPMn solutions.
+
+Main features of the Process Managment modules:
+
+* Creating customizable processes (called workflows)
+* Creation of process instances called workorders
+* Definiting workflows using steps
```

Comment:
I think we need to eventually settle on a way to write these.

`Work Flows, WorkFlows, workflows, work-flows...`